joplin/css/admin.scss

```diff
@@ -22,8 +22,8 @@
 
 .translation-tab .object .object-help {
     width: auto;
-    opacity: 1;
-    position: static;
+    // opacity: 1;
```

Comment:
do we plan on using these again in the future for something? if not let's just delete them instead of commenting them out